src/test/java/teammates/test/cases/logic/FeedbackResponsesLogicTest.java

```diff
@@ -72,33 +72,33 @@ private void testSpecialCharactersInTeamName() {
         FeedbackQuestionAttributes question = fqLogic.getFeedbackQuestion(
                 "First Session", "FQLogicPCT.CS2104", 1);
 
-        // Alice will see 4 responses
+        // Alice will see 3 responses
```

Comment:
@tshradheya  actually It was my mistake I did not change the value parameter of response 4 therefore it was creating confusion for you.
Let me elaborate why I change it to 3 as response-4 is provided by the team 1( team of alice having benny as the teammate)  when getViewableFeedbackResponsesForQuestionInSection() method is called we make a query based upon the email id (one of the argument of this method) so here feadback is given by team 1 (It may be benny or alice)   we do not have email id and if benny provide the feedback the alice will not be able to see so over scenerio boils to point that alice will for sure see only those response which is provide to his team but there may be chance she did not provide the response to other team in that case she would not be able to see response ( as current logic of teammates assume) so I reduce it to 3 (same in all places where you pointed)